src/app/record/pages/my-orcid/my-orcid.component.html

```diff
@@ -6,6 +6,8 @@
       <app-side-bar class="col l3 m8 s4"> </app-side-bar>
       <app-main class="col l9 m8 s4">
         <app-top-bar class="row"></app-top-bar>
+        <h2 class="orc-font-body col">Activities</h2>
```

Comment:
Translation is missing here

Reply:
all these translations are going to be added as part of https://trello.com/c/fHIkifjA/7226-speedup-activities-affiliations-modals-8